src/meta/src/stream/stream_graph.rs

```diff
@@ -39,46 +43,18 @@ use crate::model::FragmentId;
 use crate::storage::MetaStore;
 use crate::MetaResult;
 
-/// Id of an Actor, maybe local or global
-#[derive(Clone, Copy, Debug, Hash, Eq, PartialEq, PartialOrd, Ord)]
-enum LocalActorId {
```

Comment:
We don't need this as we're sure of the total count of actors after the scheduling, so that we can directly allocate global IDs from the ID generator and use them.